testfile2.py

```
@@ -0,0 +1,818 @@
+        )
+        return render_html_view(request, str(certificate.course_id), certificate)
+    except GeneratedCertificate.DoesNotExist as e:
+        raise Http404 from e
+
+
+@handle_500(
+    template_path="certificates/server-error.html",
+    test_func=lambda request: request.GET.get('preview', None)
+)
+@pluggable_override('OVERRIDE_RENDER_CERTIFICATE_VIEW')
+def render_html_view(request, course_id, certificate=None):  # pylint: disable=too-many-statements
```

Pylint (3.3.7)

[refactor] Too many local variables (35/15)

(R0914)